app/javascript/gobierto_data/webapp/components/Dataset.vue

```diff
@@ -414,7 +408,8 @@ export default {
       const {
         data: { data: items },
       } = response;
-      this.publicQueries = items;
+
+      this.publicQueries = !items ? [] : items
```

Comment:
You may initialize in the previous line

```suggestion
const { data: { data: items = [] } } = response;
this.publicQueries = items
```